ark/analysis/spatial_analysis.py

```diff
@@ -248,7 +249,7 @@ def create_neighborhood_matrix(all_data, dist_matrices_dict, included_fovs=None,
 
     # Error Checking
     if not np.isin(included_fovs, all_data[fov_col]).all():
-        raise ValueError("Points were not found in Expression Matrix")
```

Comment:
We do this check all the time. I think it would be worth adding a function to test_utils that does the following:
It takes in two lists of items. If any item in the first list is not present in the second list, raise a ValueError specifying which items are missing. 

Maybe there can be two options: subset and strict match, with the appropriate errors raised accordingly. You can create an issue and address this in a separate PR